dags/data_quality_checks_dag.py

```diff
@@ -0,0 +1,173 @@
+"""
+Imports local files to S3, then to CrateDB and checks several data quality properties
+
+Prerequisites
+-------------
+In CrateDB, set up tables for temporarily and permanently storing incoming data.
+See the file setup/smart_home_data.sql in this repository.
+"""
+import os
+import logging
+from airflow.utils.task_group import TaskGroup
+from airflow import DAG
+from airflow.utils.dates import datetime
+from airflow.decorators import task
+
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from airflow.providers.common.sql.operators.sql import SQLColumnCheckOperator, SQLTableCheckOperator
+from airflow.providers.amazon.aws.transfers.local_to_s3 import LocalFilesystemToS3Operator
+from airflow.providers.amazon.aws.operators.s3_copy_object import S3CopyObjectOperator
+from airflow.providers.amazon.aws.operators.s3_delete_objects import S3DeleteObjectsOperator
+
+from include.data_checks import COL_CHECKS, TABLE_CHECKS
+
+
+S3_BUCKET = os.environ.get("S3_BUCKET")
+ACCESS_KEY_ID = os.environ.get("SECRET_ACCESS_KEY")
+SECRET_ACCESS_KEY = os.environ.get("SECRET_ACCESS_KEY")
+FILE_DIR = os.environ.get("FILE_DIR")
+TEMP_TABLE = os.environ.get("TEMP_TABLE")
+TABLE = os.environ.get("TABLE")
```

Comment:
Would be nice to include a sample `.env` file with this DAG to make it as simple as possible to get the DAG running.

Reply:
I am not in favor of this as `.env` file contains sensitive data about CrateDB connection and AWS credentials.

Reply:
Although looking at this now, seems like `ACCESS_KEY_ID` should get the `access_key` not the secret from the env. Regardless, I'm approving the PR :)

Reply:
This is also fixed in the latest commit :)

Reply:
I see benefits for storing placeholder data in `.env`, but when working with the code locally, there is always a small possibility that someone pushes their own `.env` file (we would need to remove `.env` from `.gitignore`). But, in the comment section of this DAG, I can list which environment variables are required and put placeholders there.

Reply:
Ah true, better safe than sorry. Comment should be plenty!